ui/raidboss/data/07-dt/trial/zoraal-ja-ex.ts

```diff
@@ -1014,18 +1021,22 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         inside: {
           en: 'Inside Safe',
+          de: 'Innen sicher',
           fr: 'Intérieur sûr',
         },
         eastWest: {
           en: 'Inside East/West Safe',
+          de: 'Innen Osten/Westen sicher',
           fr: 'Intérieur Est/Ouest sûr',
         },
         northSouth: {
           en: 'Inside North/South Safe',
+          de: 'Innen Norden/Süden sicher',
           fr: 'Intérieur Nord/Sud sûr',
         },
         intercard: {
           en: 'Ouside Intercards Safe (Avoid Corners)',
```

Comment:
Note: EN misspelling here ("Ouside" should be "Outside")

Reply:
i will include this fix